explain.md

```diff
@@ -18,6 +68,209 @@ The user requires the appropriate [privileges](privileges.html) for the statemen
 
 ## Parameters
 
-| Parameter | Description |
+Each `EXPLAIN` statement can only include one *Mode*, but can include any number of *Options*.
+
+| Parameter | Type | Description |
+|-----------|------|-------------|
+| `PLAN` | *Mode* | Return the query plan.<br/><br/>If you do not specify a mode, the statement uses `PLAN` mode. |
+| `TYPES` | *Mode* | Include the intermediate [data types](data-types.html) CockroachDB chooses to evaluate intermediate SQL expressions. <br/><br/>`TYPES` also implies `METADATA` and `EXPRS` options.|
```

Comment:
TYPES is not a mode any more - it's now an option.


(The only remaining modes are PLAN, TRACE and DEBUG.)